Scripts/Python/Elonin.py

```diff
@@ -0,0 +1,8 @@
+from Plasma import *
```

Comment:
License block is missing.

Reply:
Added. Once we decide on a `self.id`, I'll push that along with this fix.